packages/create-spectacle/src/cli.ts

```diff
@@ -9,7 +9,7 @@ import {
   writeWebpackProjectFiles,
   writeOnePageHTMLFile
 } from './templates/file-writers';
-import { version as SPECTACLE_VERSION } from 'spectacle/package.json';
+import { version as SPECTACLE_VERSION } from '../package.json';
```

Comment:
We don't want the `create-spectacle` version, we want something like:

```suggestion
import { devDependencies: { spectacle as SPECTACLE_VERSION } } from '../package.json';
```